lightbeam/lightbeam.py

```diff
@@ -141,7 +141,21 @@ def meets_process_criteria(self, tuple):
                     or (self.newer_than and tuple[0]>self.newer_than)
                     or (len(self.resend_status_codes)>0 and tuple[1] in self.resend_status_codes)
                 )
+    
+    def _confirm_delete_op(self, endpoints, verbiage):
+        if self.config.get("force_delete", False):
+            return
+
+        endpoint_list = "\n\t • ".join(endpoints)
```

Comment:
This feature looks good, thanks @johncmerfeld.

I just wanted to ask if putting bullet chars in terminal output is typical? Probably fine, but should we consider using a dash instead? `-`

Reply:
Yeah I think you're right. I assume most consoles can handle this but probably safest to stick with ASCII; changing now